pyuvdata/uvdata/tests/test_ms.py

```diff
@@ -1063,3 +1075,16 @@ def test_flip_conj_multispw(sma_mir, tmp_path):
     sma_mir.filename = ms_uv.filename = None
 
     assert sma_mir == ms_uv
+
+
```

Comment:
I'm _think_ this is the last warning that needs to be caught:

```suggestion
@pytest.mark.filterwarnings("ignore:"Writing in the MS file that the units of the data")
```